File: Beecrowd/Beginner/2003 - 2176/2003 - Sunday Morning/C++/2003_SundayMorning.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int hour, minute;
    char semicolon;

    while (cin >> hour >> semicolon >> minute) {
        int delay = 0;

        if (hour >= 7) {
            delay = (hour - 7) * 60 + minute;
        }

        cout << "Atraso maximo: " << delay << endl;

    }

    return 0;
}

File: Beecrowd/Beginner/2003 - 2176/2003 - Sunday Morning/Python/2003_SundayMorning.py
while True:
    try:
        hour, minute = map(int, input().split(":"))
        delay = 0

        if hour >= 7:
            delay = (hour - 7) * 60 + minute

        print(f"Atraso maximo: {delay}")

    except EOFError:
        break
